<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>201</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>201</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>201</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="PlayButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Play!</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ComboGameMode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Local game</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Online game</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="LabelServerIP">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ServerIP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineServerIP">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="ComboBoardSize">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>9x9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13x13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19x19</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="LabelBoardSize">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BoardSize</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelHandicap">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Handicap</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="SpinHandicap">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QComboBox" name="ComboRole">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Hold</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Join</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="LabelRole">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Role</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ComboStone">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>65</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Black</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>White</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="LabelStone">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Stone</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelKomi">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Komi</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="SpinKomi">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>9.500000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.500000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
    <property name="value">
     <double>6.500000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="LabelTime">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Time(Hour)</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="SpinTime">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>1440.000000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonLoadConfig">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>LoadConfig</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonSaveConfig">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>SaveConfig</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
